packages/react/src/types/component-props.ts

```diff
@@ -201,50 +225,70 @@ export interface ReactFlowProps<NodeType extends Node = Node, EdgeType extends E
+  /**
+   * This event handler is called when panning or zooming viewport movement stops.
+   * If the movement is not user-initiated, the event parameter will be `null`.
+   */
   onMoveEnd?: OnMoveEnd;
-  /** This event handler gets called when a user changes group of selected elements in the flow */
+  /** This event handler gets called when a user changes group of selected elements in the flow. */
   onSelectionChange?: OnSelectionChangeFunc<NodeType, EdgeType>;
-  /** This event handler gets called when user scroll inside the pane */
+  /** This event handler gets called when user scroll inside the pane. */
   onPaneScroll?: (event?: WheelEvent) => void;
-  /** This event handler gets called when user clicks inside the pane */
+  /** This event handler gets called when user clicks inside the pane. */
   onPaneClick?: (event: ReactMouseEvent) => void;
-  /** This event handler gets called when user right clicks inside the pane */
+  /** This event handler gets called when user right clicks inside the pane. */
   onPaneContextMenu?: (event: ReactMouseEvent | MouseEvent) => void;
-  /** This event handler gets called when mouse enters the pane */
+  /** This event handler gets called when mouse enters the pane. */
   onPaneMouseEnter?: (event: ReactMouseEvent) => void;
-  /** This event handler gets called when mouse moves over the pane */
+  /** This event handler gets called when mouse moves over the pane. */
   onPaneMouseMove?: (event: ReactMouseEvent) => void;
-  /** This event handler gets called when mouse leaves the pane */
+  /** This event handler gets called when mouse leaves the pane. */
   onPaneMouseLeave?: (event: ReactMouseEvent) => void;
   /**
-   * Distance that the mouse can move between mousedown/up that will trigger a click
+   * Distance that the mouse can move between mousedown/up that will trigger a click.
    * @default 0
    */
   paneClickDistance?: number;
   /**
-   * Distance that the mouse can move between mousedown/up that will trigger a click
+   * Distance that the mouse can move between mousedown/up that will trigger a click.
    * @default 0
    */
   nodeClickDistance?: number;
-  /** This handler gets called before the user deletes nodes or edges and provides a way to abort the deletion by returning false. */
+  /**
+   * This handler is called before nodes or edges are deleted, allowing the deletion to be aborted
+   * by returning `false` or modified by returning updated nodes and edges.
+   */
   onBeforeDelete?: OnBeforeDelete<NodeType, EdgeType>;
   /**
    * Custom node types to be available in a flow.
    *
-   * React Flow matches a node's type to a component in the nodeTypes object.
+   * React Flow matches a node's type to a component in the `nodeTypes` object.
+   * @TODO check if @default is correct
+   * @default {
+   *   input: InputNode,
+   *   default: DefaultNode,
+   *   output: OutputNode,
+   *   group: GroupNode
+   * }
```

Comment:
```suggestion
   * @default {
   *   input: InputNode,
   *   default: DefaultNode,
   *   output: OutputNode,
   *   group: GroupNode
   * }
```

I copied `@default` from the reactflow API reference. Can you agree on whether it is correct?